Source/WebCore/layout/formattingContexts/inline/InlineLine.h

```diff
@@ -28,6 +28,7 @@
 #include "InlineDisplayBox.h"
 #include "InlineItem.h"
 #include "InlineTextItem.h"
+#include "RenderStyleInlines.h"
```

Comment:
Is this include needed ?

Reply:
Yes, because RenderStyle::letterSpacing moved into RenderStyleInlines.h (from RenderStyle.cpp), and that means this line at the bottom of the page won't compile:
https://searchfox.org/wubkat/source/Source/WebCore/layout/formattingContexts/inline/InlineLine.h#370